src/inmanta/agent/executor.py

```diff
@@ -212,7 +212,7 @@ class ResourceInstallSpec:
     """
     This class encapsulates the requirements for a specific resource type for a specific model version.
 
-    :ivar resource_type: fully qualified name for this resource type e.g. std::File
+    :ivar resource_type: fully qualified name for this resource type e.g. std::testing::NullResource
```

Comment:
i'm not sure using "testing" in an example is nice but I don't know what else to put here.